src/pudl/settings.py

```diff
@@ -310,9 +329,7 @@ class Ferc1ToSqliteSettings(GenericDatasetSettings):
         years: List of years to validate.
     """
 
-    working_partitions: ClassVar = {
-        "years": list(pc.WORKING_PARTITIONS["ferc1"]["years"])
-    }
+    working_partitions: ClassVar = Ferc1Settings.working_partitions
```

Comment:
Hmm, this is a little different/weird, since these settings pertain to the original extraction from the Visual FoxPro DB. But the working years should always be the same between the two. Would it also make sense to just embed the `DataSource` here rather than grabbing the stored `working_partitions` from the `Ferc1Settings` which were set using the `DataSource` values?

Reply:
`working_partitions` is coming directly from the `DataSource`, but that approach is a bit indirect and confusing, I can go ahead and switch to using the `DataSource` directly (same for comment below)